Makefile

```diff
@@ -363,7 +363,7 @@ cli: cli-build
 	chmod u+w $(GOPATH)/bin/roxctl
 
 cli-linux: build-prep
-	RACE=0 CGO_ENABLED=0 GOOS=linux $(GOBUILD) ./roxctl
+	RACE=0 CGO_ENABLED=0 GOOS=linux GOARCH=amd64 $(GOBUILD) ./roxctl
 
 cli-darwin: build-prep
```

Comment:
Do we need to worry about making an amd64 and/or arm64 for Darwin as well?

Reply:
Oh actually, Ross I see your point. Cong we should add AMD64 to darwin build as well

Reply:
Darwin is our local dev machine, it would be convenient if we can use it with local architecture. If I change it to arm64, we will be running in emulated mode. :(